cpu/stm32f0/ldscripts/stm32f030cc.ld

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I think this file is not needed anymore (ROM and RAM size are computed directly from cpu name)